src/StatusQ/Controls/StatusBaseButton.qml

```diff
@@ -100,6 +100,7 @@ Rectangle {
 
     MouseArea {
         id: sensor
+        anchors.centerIn: parent
         width: layout.width + statusBaseButton.leftPadding + statusBaseButton.rightPadding
         height: layout.height + statusBaseButton.topPadding + statusBaseButton.bottomPadding
```

Comment:
Wouldn't `anchors.fill: parent` and removing width/height do the same job? :)

Reply:
you're right, updated it a bit so that the entire button is clickable as well